### ISSUE TITLE
Phthiraptera - Journal Keywords -no automatic indexing for new keywords

### ISSUE THREAD

**Issue by scratchpads** [NaturalHistoryMuseum/scratchpads2#5949]
- Added by [Yokb](http://phthiraptera.info/user/29)
- Scratchpad: [Phthiraptera.info](http://phthiraptera.info)

### Description:

When I add new keywords (i.e. that has not previously appeared in the Jouranl Kewywords) to the datarecord. It does not seemed to appear in the selection list under Journak Keywords under the LITERATURE page. Nor when I click onto the keyword in the datarecord, it would stated no record found. You said that there should be automatically created or index. So appreciate can you look into this

**Comment by jrdh** [comment]
I just gave this a go locally in my development environment and it worked fine. Typically I had to wait up to 2 mins for the keyword change to come through on the search page (for both addition and deletion) which sounds about right given I'm pretty sure the solr indexes are updated with entity changes using a queue which is processed periodically. Perhaps this update process is taking longer on the Phthiraptera site given how many entities there are, however, I'm surprised you're not seeing the keywords changes come through at all. I did just try adding a new keyword to [this entry](http://phthiraptera.info/node/94668) and it worked fine (I added `joshtest` which I've since removed) - the keyword was visible on the page, in the search facets and when I searched for the keyword I got the correct results. I'll see if I can chat to you about it in person tomorrow if you're in, otherwise, it'd be good to know which journal were you updating and which keyword did you add that didn't come through?
